source/blog/2016-08-22-last-week-in-manageiq.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Last Week in ManageIQ: "
+date: 2016-08-22 07:30 UTC
+author: juliancheal
+comments: true
+published: true
+---
+
+What ho! Welcome to another jolly edition of Last Week in ManageIQ. [Julian](https://juliancheal.co.uk) here, hailing from the Providers team. Let's put the kettle on and take a look over what's happened this last week in ManageIQ.
+
+## Featured
+
+We've had a spiffing week with [138 pull requests merged][PRs merged last week] from our marvellous contributors. ManageIQ always welcomes new contributors, we have a great list of
```

Comment:
The count may be off. I'm seeing 147 with this url. 

Though I know we've had a number of discussions around how to count these 😄  - So please ignore if I'm behind on that front.